source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -350,6 +412,28 @@ def next_passage_for_route_point_test(mock_good_response):
         assert passages[1].is_real_time
 
 
+def next_passage_for_route_point_with_direction_type_test(mock_direction_type_is_forward_response):
+    """
+    test the whole next_passage_for_route_point with direction_type parameter
+    mock the http call to return a good response, we should get some next_passages
+    """
+    sytral = Sytral(id='tata', service_url='http://bob.com/')
+
+    mock_requests = MockRequests(
+        {'http://bob.com/?stop_id=42&direction_type=forward': (mock_direction_type_is_forward_response, 200)}
```

Comment:
So we don't use line_code but we use direction_type... This is quite weird to ask for all "forward" routes passing at a stop, no?